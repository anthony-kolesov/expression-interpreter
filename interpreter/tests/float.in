out 9.7
print "\n"
out 4.5 + 2*10 + 3*( 5 + 1 )
print "\n"
out 4 + 2.1*10 + 3*( 5 + 1 )
print "\n"

