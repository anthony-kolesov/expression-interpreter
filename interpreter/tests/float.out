9.700000
42.500000
43.000000
